bin/tests/check_dl_cv.sh

```diff
@@ -603,5 +603,61 @@ fi
 rm -rf ./tests/logs/_tests_cv_segmentation
 
 
+################################  pipeline 23  ################################
+echo 'pipeline 23'
+EXPDIR=./tests/_tests_cv_classification_hydra
+LOGDIR=./tests/logs/_tests_cv_classification_hydra
+LOGFILE=${LOGDIR}/checkpoints/_metrics.json
+
+PYTHONPATH=./examples:./catalyst:${PYTHONPATH} \
```

Comment:
you need to remove `./catalyst/` to resolve typing issue, Catalyst also has `catalyst/typing.py`
```suggestion
PYTHONPATH=./examples:.:${PYTHONPATH} \
```

Reply:
Thanks. It works now! =)